R/mash_wrapper.R

```diff
@@ -923,22 +928,23 @@ mash_pipeline <- function(mash_input, alpha, residual_correlation = NULL, uncons
       vhat <- diag(rep(1, condition_num))
     }
   } else {
-    vhat <- estimate_null_correlation_simple(mash_set_data(mash_input$null.b,
+    vhat <- mashr::estimate_null_correlation_simple(mashr::mash_set_data(mash_input$null.b,
       Shat = mash_input$null.s,
       alpha, zero_Bhat_Shat_reset = 1000
     ))
   }
 
   # mash data Fit mixture model using udr package
-  mash_data <- mash_set_data(mash_input$strong.b,
+  mash_data <- mashr::mash_set_data(mash_input$strong.b,
     Shat = mash_input$strong.s, V = vhat,
     alpha, zero_Bhat_Shat_reset = 1000
   )
   # Canonical matrices
-  U.can <- cov_canonical(mash_data)
+  U.can <- mashr::cov_canonical(mash_data)
 
   # Penalty strength
   lambda <- ncol(mash_input$strong.z)
+  # NOTE: Please change this to use flashier + ED instead of UDR
```

Comment:
@danielnachun this is very considerate -- could you use FIXME instead of NOTE? I typically do that then at some point search for all FIXME string in the code repo to see what needs extra work